core/src/main/scala/org/apache/spark/sql/execution/ColumnarBroadcastExchangeExec.scala

```diff
@@ -113,9 +113,8 @@ class ColumnarBroadcastExchangeExec(mode: BroadcastMode, child: SparkPlan)
         hashRelationKernel.build(
           hash_relation_schema,
           Lists.newArrayList(hash_relation_expr),
-          null,
-          true,
-          SparkMemoryUtils.globalMemoryPool())
```

Comment:
@zhztheplayer need to revert this changes here.  seems conflict with new BHJ code.

Reply:
OK. Is this the key to fix error shown in #4 ?